### PR TITLE
Add glances

### PR DESCRIPTION
**Pull request by jakirkham**
Adds a recipe to build glances on all platforms. Based on the conda recipes with some modifications to make it fit nicely at conda-forge.

### CODE REVIEW

File: recipes/glances/bld.bat [hunk 1]
```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install
```
Comment (ocefpaf) [inline recipes/glances/bld.bat]: You can use `--single-version-externally-managed --record=record.txt` here too.

Comment (jakirkham) [inline recipes/glances/bld.bat]: Good call. Sorry, missed that. Adding...

Comment (jakirkham) [inline recipes/glances/bld.bat]: Fixed.

Comment (ocefpaf) [inline recipes/glances/bld.bat]: You are fast!

Comment (jakirkham) [inline recipes/glances/bld.bat]: What can I say, I am pretty excited about getting things into conda-forge.

**Comment by jakirkham** [comment]
Tests are passing. :)

Any more comments?

**Comment by jakirkham** [comment]
@nicolargo, just letting you know we have added `glances` to conda-forge. This will allow users to easily install this with conda (`conda install -c conda-forge glances`). The maintenance burden here is light as all testing and releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe in [this repo]( https://github.com/conda-forge/glances-feedstock ). If you have any interest in being added as a maintainer (to make new releases for instance), I can happily add you to the maintainers list.

**Comment by nicolargo** [comment]
@jakirkham thanks for the head up. Can i add you to the Glances AUTHORS file (https://github.com/nicolargo/glances/blob/master/AUTHORS) as the official maintener for the Conda package ? If yes can you send me your public email address in order to be added to the maintener mailing list (only one mail when a new Glances version is released) ?

**Comment by jakirkham** [comment]
No problem. Sure, would you like me to submit a PR?

**Comment by nicolargo** [comment]
Yes, on the DEVELOP branch please ;)

**Comment by jakirkham** [comment]
PR added here ( https://github.com/nicolargo/glances/pull/818 ). Email on the way.

**Comment by nicolargo** [comment]
Yes, i add you email to the mailing list.

**Pull request merged**